custom/.gitpod.yml

```diff
@@ -0,0 +1,71 @@
+image: axonasif/gitpod-base-node-vnc:latest # Locally available as `.gitpod.Dockerfile`
+
+github:
+  prebuilds:
+    master: true
+    pullRequests: true
+
+gitConfig:
+  pull.rebase: "false"
+  core.hooksPath: ".githooks"
+
+tasks:
+  # Re(from meeting notes): Is it possible to create workspaces that are specific to a part of the codebase?
```

Comment:
>  Re(from meeting notes): Is it possible to create workspaces that are specific to a part of the codebase?
  e.g. the front-end team don’t need access to the whole application, just the front end.
  Having a multi-configuration per repo would be ideal. Apparently, Codespaces has this feature.

We have https://www.gitpod.io/docs/multi-repo-workspaces, but it might not be what you're looking for. Perhaps our webapp team can look into implementing such a feature.

> Re(from meeting notes): Is there anything in their set-up/configuration that they can do to speed up their workspace boot-up times?

What if you guys split up the frontend and backend into two repositories(if feasible, considering that you utilize monorepos)?
Later you can remove the build steps that is unrelated(backend) to frontend and vise versa from the repo specific `.gitpod.yml`
Also, you may then use git submodules for each other for ease of access from one repo to another.